packages/cosmic-proto/src/helpers.ts

```diff
@@ -43,6 +47,7 @@ type ResponseMap = {
   '/cosmos.bank.v1beta1.QueryAllBalancesRequest': '/cosmos.bank.v1beta1.QueryAllBalancesResponse';
   '/cosmos.staking.v1beta1.MsgDelegate': '/cosmos.staking.v1beta1.MsgDelegateResponse';
   '/ibc.applications.transfer.v1.MsgTransfer': '/ibc.applications.transfer.v1.MsgTransferResponse';
+  '/cosmos.staking.v1beta1.MsgUndelegate': '/cosmos.staking.v1beta1.MsgUndelegateResponse';
```

Comment:
I didn't really grok this when I did `undelegate` in `stakingAccountKit`.

And I guess review didn't catch it.